extensions/adobe/experience/adcloud/search/adgroup-all.schema.json

```diff
@@ -0,0 +1,70 @@
+{
+    "meta:license": [
+        "Copyright 2019 Adobe Systems Incorporated. All rights reserved.",
+        "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+        "you may not use this file except in compliance with the License. You may obtain a copy",
+        "of the License at https://creativecommons.org/licenses/by/4.0/"
+    ],
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "$id": "https://ns.adobe.com/experience/adcloud/search/adgroup-all",
+    "title": "Adobe AdCloud Search Adgroup Full Extension",
+    "type": "object",
+    "meta:extensible": true,
+    "meta:abstract": true,
+    "meta:intendedToExtend": [
+        "https://ns.adobe.com/xdm/context/profile-adcloud"
+    ],
+    "description": "Adobe AdCloud Search Adgroup Full Extension. Contains all Solution added fields.",
+    "definitions": {
+        "adgroup-all":{
+            "properties": {
+                "adgroup": {
+                    "title": "Adobe Ad Cloud Search Adgroup Top Level Fields",
+                    "$ref": "https://ns.adobe.com/experience/adcloud/search/advertisingattributes",
+                    "description": "Adobe Ad Cloud Top Level Attributes for Adgroups."
+                },
+                "campaignInternalId": {
+                    "title": "Campaign Unique Identifier",
+                    "type": "integer",
+                    "description": "Campaign unique identifier for the campaign this adgroup belongs to. Use for joining with campaigns."
+                },
+                "bidLevel": {
+                    "title": "Bid Level",
+                    "type": "string",
+                    "description:": "What determines the bid for all ads belonging to this adgroup.",
+                    "meta:enum": {
+                        "gender": "Gender",
+                        "placement": "Placement",
+                        "vertical": "Vertical",
+                        "age_range": "Age Range",
+                        "keyword": "Keyword",
+                        "adgroup": "Adgroup",
+                        "user_interest_and_list": "User Interest and List"
+                    }
+                },
+                "mobileBidAdjustment": {
+                    "title": "Mobile Bid Adjustment",
+                    "type": "integer",
```

Comment:
if it's a percentage, shouldn't this be a number and not an integer?

Reply:
Mobile Bid Adjustment is a deprecated field and is not required in schema.